JakinChan200/CS180_Roald
# #14 Store CSV data into appropriate data structures

## Overview
Adds a function that converts the data from the temporary data structure from https://github.com/JakinChan200/CS180_Roald/issues/11 into a proper array.
Temporarily called in index.js on server startup for testing purposes.
Modified some CSV files to remove columns that are not necessary for our data analysis.
Countries called in index.js are CA and DE for testing purposes.

## Materials
- [Issue] https://github.com/JakinChan200/CS180_Roald/issues/14

## Pictures
N/A

## Manual Testing Guide
- Start the server.
- The output is exactly the same, but the output stems from an array rather than the temporary object data structure used before.

Pull requests should receive approval before merge.
